shell/common/shell_unittests.cc

```diff
@@ -2843,14 +2851,17 @@ TEST_F(ShellTest, IgnoresInvalidMetrics) {
   RunEngine(shell.get(), std::move(configuration));
 
   task_runner->PostTask([&]() {
-    shell->GetPlatformView()->SetViewportMetrics({0.0, 400, 200, 22, 0});
+    shell->GetPlatformView()->SetViewportMetrics(kDefaultViewId,
+                                                 {0.0, 400, 200, 22, 0});
     task_runner->PostTask([&]() {
-      shell->GetPlatformView()->SetViewportMetrics({0.8, 0.0, 200, 22, 0});
+      shell->GetPlatformView()->SetViewportMetrics(kDefaultViewId,
+                                                   {0.8, 0.0, 200, 22, 0});
       task_runner->PostTask([&]() {
-        shell->GetPlatformView()->SetViewportMetrics({0.8, 400, 0.0, 22, 0});
+        shell->GetPlatformView()->SetViewportMetrics(kDefaultViewId,
+                                                     {0.8, 400, 0.0, 22, 0});
         task_runner->PostTask([&]() {
           shell->GetPlatformView()->SetViewportMetrics(
-              {0.8, 400, 200.0, 22, 0});
+              kDefaultViewId, {0.8, 400, 200.0, 22, 0});
```

Comment:
Is there a test that checks that setting the viewport metrics for a nonexistent view is ignored? If not, this may be a good place for that

Reply:
Done!